packages/react/src/components/MultiSelect/index.js

```diff
@@ -4,11 +4,16 @@
  * This source code is licensed under the Apache-2.0 license found in the
  * LICENSE file in the root directory of this source tree.
  */
+import { deprecateFieldOnObject } from '../../internal/deprecateFieldOnObject';
 
 import MultiSelect from './MultiSelect';
 import FilterableMultiSelect from './FilterableMultiSelect';
 
 FilterableMultiSelect.displayName = 'MultiSelect.Filterable';
 MultiSelect.Filterable = FilterableMultiSelect;
 
+if (__DEV__ && !process.env.JEST_WORKER_ID) {
```

Comment:
is the `JEST_WORKED_ID` check to see if we're in a test? If so, I think we can use that `mock` strategy mentioned above for any tests for this method instead

Reply:
Fixed